music_assistant/server/providers/snapcast/__init__.py

```diff
@@ -109,7 +109,7 @@ async def get_config_entries(
     values: the (intermediate) raw values for config entries sent with the action.
     """
     returncode, output = await check_output("snapserver -v")
-    snapserver_present = returncode == 0 and "snapserver v0.27.0" in output.decode()
+    snapserver_present = returncode == 0 and "snapserver v0.2" in output.decode()
```

Comment:
I would not do that - we need actually the version to be minimum 0.27 so what I would do here is parse the version and compare it. So first check if the returncode is 0. Next step is parse the version.